addons/vz99/functions/fnc_handleFired.sqf

```diff
@@ -0,0 +1,63 @@
+/*
+ * Author: PabstMirror
+ *
+ *
+ * Arguments:
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [] call
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_mortarVeh", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile"];
+TRACE_7("",_mortarVeh, _weapon, _muzzle, _mode, _ammo, _magazine, _projectile);
+
+if (!local _mortarVeh) exitWith {};
+
+//Remove the empty magazine, so a new one will load correctly
+private _loadedEmpty = _mortarVeh magazinesTurret [0];
+if (!(_loadedEmpty isEqualTo [])) then {
+    TRACE_1("removing empty mag", _loadedEmpty);
+    _mortarVeh removeMagazinesTurret [(_loadedEmpty select 0), [0]];
+};
+
+private _fuze = getText (configFile >> "CfgMagazines" >> _magazine >> QGVAR(fuze));
+if (_fuze == "") exitWith {};
+
+private _detonationHeight = switch (true) do {
```

Comment:
This if else seems to logically fit better here.
(Also file header not filled in)